src/common/containers/ProjectList/index.jsx

```diff
@@ -54,7 +54,8 @@ class ProjectListContainer extends Component {
       const projectURL = `/projects/${project.name}`
       const memberHandles = (project.members || []).map(member => {
         const memberURL = `/users/${member.handle}`
-        return <Link key={member.handle} to={memberURL}>{member.handle}</Link>
+        const linkKey = `${project.cycle.cycleNumber}-${member.handle}`
```

Comment:
@sdweber422: in theory, it's possible for a user to be in multiple projects in the same cycle. instead, let's make the key this: `PROJECT_NAME-MEMBER_HANDLE`.